dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2358,3 +2358,23 @@ def test_filter_nonpartition_columns(
     df_read = ddf_read.compute()
     assert len(df_read) == len(df_read[df_read["time"] < 5])
     assert df_read["time"].max() < 5
+
+
+@pytest.mark.parametrize("dtype", ["Int64", "str"])
+def test_pandas_metadata_nullable_pyarrow(tmpdir, dtype):
```

Comment:
I think this needs to be skipped for pandas<1.0.0, which added string dtype and the Arrow interface for reconstructing extension dtypes from parquet.

Reply:
I am currently skipping for `< LooseVersion("1.0.0")` below - Are you saying I should use a different mechanism to skip?

Reply:
Sorry, I think I missed that. This looks good.